test/kata3.numberToReversedDigits.test.js

```diff
@@ -1,5 +1,7 @@
 const { numberToReversedDigits } = require("../src");
 
 describe("numberToReversedDigits", () => {
-  test("returns a reversed array of the number's digits", () => {});
+  it("returns a reversed array of the number's digits", () => {
```

Comment:
Could some more tests be added here? Always good practice to cover the different possibilities to make sure they are accounted for